Makefile

```diff
@@ -12,9 +12,9 @@
 
 SYSROOT	= sysroot
 INCLUDE = $(SYSROOT)/usr/include
-CPP_SRC = $(shell find kernel/ -name "*.cpp")
-ATT_SRC = $(shell find kernel/ -name "*.s")
-HEADERS = $(shell find $(INCLUDE) -name "*.hpp" -name "*.h")
+CPP_SRC = $(shell find kernel -name "*.cpp")
+ATT_SRC = $(shell find kernel -name "*.s")
+HEADERS = $(shell find $(INCLUDE) -name "*.hpp")
```

Comment:
Do these lines need to be included in this PR?

Reply:
Technically no. I should have changed that on dev.

Reply:
Are you going to change this?

Reply:
Ah, I forgot. I can change it back.

Reply:
Pushed.